### DISCUSSION TITLE
Problems installing Pages using Joomlatools Console

### DISCUSSION THREAD

**Discussion by pun2006** [joomlatools/joomlatools-pages#768] [General]
Meant to output the content of hello page.

#### Steps to reproduce the behavior

1.  joomla site:create testsite 
2. composer.phar require joomlatools/pages --ignore-platform-reqs
3. Complete the instructions from quickstart guide

**Comment by johanjanssens** [comment]
Hello @pun2006 

I'm afraid based on the information you provided there is not much I can help you with. Without any indication of an error, or where you are getting stuck. 

This approach you outline requires that you have our Joomlatools Console installed, and or are running our Joomlatools Vagrant box. This is a more complex flow. 

If you have problems with one of the more advanced installation methods, try to use the most simple approach by installing Pages using the Joomla installer as documented here: https://github.com/joomlatools/joomlatools-pages/wiki/Installation#installing-with-the-joomla-installer
